.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!-- Thanks for filing an issue! Before hitting the button, please answer these questions.-->
 
-**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see http://kubernetes.io/docs/troubleshooting/.):
+**Is this a request for help?** (If yes, you should use our troubleshooting guide and community support channels, see https://kubernetes.io/docs/tasks/debug-application-cluster/troubleshooting/.):
```

Comment:
@xiangpengzhao @supereagle I'm okay with it either way. It doesn't impact the link at all, so the period really doesn't matter. If you decide to remove it @xiangpengzhao, I don't mind re-LGTMing it.